core/src/main/java/com/yahoo/imapnio/async/internal/ImapAsyncSessionImpl.java

```diff
@@ -168,7 +181,7 @@ private String getNextTag() {
 
         final String tag = getNextTag();
         final StringBuilder sb = new StringBuilder(tag).append(ImapClientConstants.SPACE).append(command.getCommandLine());
-        if (logger.isDebugEnabled()) {
+        if (isDebugEnabled()) {
             logger.debug(CLIENT_LOG_REC, sessionId, new StringBuilder(tag).append(ImapClientConstants.SPACE).append(command.getLogLine()).toString());
         }
         sendRequest(sb.toString());
```

Comment:
why not logging sb.toString(), why recreating the sting to be logged? would like to log the actual data being sent to the server, why not log in the send request and receive response

Reply:
Done.